### DISCUSSION TITLE
Putting live url in the repo

### DISCUSSION THREAD

**Discussion by mohamedelbachir** [Grand-Nord-Developpers-Community/website#3] [General]
Hi, @OumarouSandaSouley , i think it will be suit to have live URL in the repository description, in order to let anyone preview change on the actual website , also we will aware of the fact everyone should have his branch and we leave just `main` branch as the **production branch**

**Comment by OumarouSandaSouley** [comment]
Greatings @mohamedelbachir 
For the branch issues, I think i have mentionned clearly in the Code of Conduct that everybody should create its own branch and I will as an administrator after verifying and testing synchronise it with the main branch to avoid merge conflict.
And talking about the live preview url i will host the main branch on Vercel,It's a Good Idea !!!
Thanks

**Reply by mohamedelbachir** [reply]
ok , thanks please i don't undersand the use of `sections` folder since we have `components` folders

**Comment by OumarouSandaSouley** [comment]
The components folder holds necessary reusable components ans sections are such of a components but large one that are parts of the page.

And for the live URL I have just noticed that to host Organizations repo in Vercel you will need Team based account witch is 20$/mo minimum so......

**Reply by mohamedelbachir** [reply]
Ok, i got it , i also encountered this behaviour when i tried to deploy organization repository it seems to be not free and i think they just give one week for free trial, and that is not enought. Maybe by using `github action` we should endup by having a possibility to watch any changes in the organization repository and apply to any personnal repository like updating the forking and maybe with this we can bypass it. But it is enough of work we just have to find a way to have at least one preview link. what do you think about it @2zalab

**Comment by OumarouSandaSouley** [comment]
For now it's better to leave it as it is and if someone needs a preview, he better clone the repo and test it by it's own.

**Reply by mohamedelbachir** [reply]
ok
